src/lang/qbittorrent_ar.ts

```diff
@@ -84,7 +84,7 @@
     <message>
         <location filename="../gui/aboutdialog.cpp" line="67"/>
         <source>An advanced BitTorrent client programmed in C++, based on Qt toolkit and libtorrent-rasterbar.</source>
-        <translation>عميل بت تورنت متقدم مُبرمج بلغة ++C ، باستخدام أدوات كيو تي البرمجية و ليبتورنت-ريستربار.</translation>
+        <translation>عميل بت تورنت متقدم مُبرمج بلغة +&#8288;+&#8288;C ، باستخدام أدوات كيو تي البرمجية و ليبتورنت-ريستربار.</translation>
```

Comment:
Why did you move ++ totally to the left and C far away to the middle?

Reply:
I did not move it, jumped there as soon as I started typing, and here is what looks like on the screen for me:
<img width="559" alt="image" src="https://user-images.githubusercontent.com/10462006/157973345-de2e8688-7ed0-469e-a476-c4f6cfce6e2a.png">